src/prob/opf_capc.jl

```diff
@@ -0,0 +1,132 @@
+"Solve OPF with capacitor control"
+function solve_mc_opf_capc(data::Union{Dict{String,<:Any},String}, model_type::Type, solver; kwargs...)
+    return solve_mc_model(data, model_type, solver, build_mc_opf_capc; kwargs...)
+end
+
+
+"Constructor for capcontrol OPF"
```

Comment:
should correct to follow new conventions for docstring format that include the function signature